src/Features/ExternalAccess/Copilot/GenerateDocumentation/CopilotDocumentationCommentProposalWrapper.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Immutable;
+using Microsoft.CodeAnalysis.DocumentationComments;
+
+namespace Microsoft.CodeAnalysis.ExternalAccess.Copilot
+{
+    internal sealed class CopilotDocumentationCommentProposalWrapper
+    {
+        private readonly DocumentationCommentProposal _documentationCommentProposal;
+        private readonly ImmutableArray<CopilotDocumentationCommentProposedEditWrapper> _wrappedProposedEdits;
+
+        public CopilotDocumentationCommentProposalWrapper(DocumentationCommentProposal documentationCommentProposal)
+        {
+            _documentationCommentProposal = documentationCommentProposal;
+            _wrappedProposedEdits = _documentationCommentProposal.ProposedEdits.SelectAsArray(e => new CopilotDocumentationCommentProposedEditWrapper(e));
+
+        }
+
+        public string SymbolToAnalyze => _documentationCommentProposal.SymbolToAnalyze;
```

Comment:
I'm trying to understand why we have one SymbolToAnalyze per proposal wrapper.

Reply:
I see, seem like there is one `CopilotDocumentationCommentProposalWrapper` per tag

Reply:
the symboltoanalyze is the code being analyzed to create each piece of the comment. No need to pass it along to every proposededit since it's the same for all.